jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  modulePathIgnorePatterns: ["<rootDir>/spec/regression"],
+};
```

Comment:
We don't want the jest tests to pick up the playwright tests, so I'm adding this configuration.

This PR wasn't responsible for the issue, but while testing it I noticed that `npm run test` was failing, because it was including the playwright regression tests.